apps/omg/lib/omg/typed_data_hash/tools.ex

```diff
@@ -95,12 +95,16 @@ defmodule OMG.TypedDataHash.Tools do
       |> Stream.concat(Stream.cycle([empty_output_hash]))
       |> Enum.take(Transaction.Payment.max_outputs())
 
+    tx_data = ABI.TypeEncoder.encode_raw([0], [{:uint, 256}])
+    metadata = metadata || <<0::256>>
```

Comment:
I'm nitpicking, but the second item in the list below doesn't get its own variable, not sure these two deserve it 😛 They should either all get intermediate variables, or just all not get any. 

```
ABI.TypeEncoder.encode_raw([:binary.decode_unsigned(plasma_framework_tx_type)], [{:uint, 256}])
```